src/record/record-transition.js

```diff
@@ -136,10 +153,19 @@ RecordTransition.prototype.add = function (socketWrapper, version, message) {
   }
 
   if (message.action === C.ACTIONS.PATCH) {
-    if (message.data.length !== 4) {
+    if (!(message.data.length === 5 || message.data.length === 4)) {
```

Comment:
can we do 

`message.data.length !== 5 && message.data.length !== 4 `

instead?